.env.template

```diff
@@ -1,6 +1,6 @@
 HUNDRED_EYES_PROJECT_NAME=100eyes
 POSTGRES_PASSWORD=
-POSTGRES_USER=
+POSTGRES_USER=postgres
```

Comment:
I think this is not necessary, as it needs to be edited by the developer. But I'm super OK to merge in the other changes as long as they do not break our builds.

Reply:
on my env it took "root" as default  and then exploded as there is no root user set on pgsql